### PR TITLE
Chinese - Chapter 1 finished

### PR DESCRIPTION
**Pull request by 1375626371**
Translation to Chinese: chapter 1 finished
@petrichor1122 and @zhlhyx completed their respective sections and we made the submission together.

### CODE REVIEW

File: chapters/zh/_toctree.yml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    title: 序列到序列模型
+  - local: chapter1/8
+    title: 偏见和局限性
+  - local: chapter1/9
+    title: 总结
+  - local: chapter1/10
+    title: 章末小测验
+    quiz: 1
+
+- title: 2. Using 🤗 Transformers
```
Comment (lewtun) [inline chapters/zh/_toctree.yml:30]: We can remove all these sections that haven't been translated yet

File: chapters/zh/_toctree.yml [hunk 2]
```diff
@@ -0,0 +1,173 @@
+- title: 0. 准备
```
Comment (lewtun) [inline chapters/zh/_toctree.yml:1]: We can remove Chapter 0 from the table of contents since it hasn't been translated yet

**Review by lewtun — COMMENTED** [review]
Thank you all for translating a whole chapter 🤯 ! This looks great and the only thing we need to do is tweak the table of contents 

To test we can build the docs in the CI, could you please add `zh` to the `languages` field in the `build_documentation.yml` and `build_pr_documentation.yml` files? In other words, something like this:

```
 languages: ar bn en es fa fr he ko pt ru th tr zh
```

**Comment by 1375626371** [comment]
We made the required changes. @lewtun

**Review by lewtun — APPROVED** [review]
Thank you for adding the final tweaks! This looks great and I'll merge when the CI passes 😍

**Comment by 1375626371** [comment]
Sorry, we have some formatting errors, which we have checked and fixed.

**Comment by lewtun** [comment]
Nice, the content renders great!

Screen Shot 2022-04-13 at 11 15 29

File: chapters/zh/chapter1/8.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Bias and limitations
```
Comment (lewtun) [inline chapters/zh/chapter1/8.mdx:1]: While looking at the rendered output I noticed this is still in English. Would it make sense to translate it too?

**Comment by 1375626371** [comment]
image
We passed the code inspection locally, but did not pass the code quality inspection of github, what should we do?@lewtun

**Comment by lewtun** [comment]
> We passed the code inspection locally, but did not pass the code quality inspection of github

Don't worry - I fixed it on my side :)

File: chapters/zh/chapter1/8.mdx (hunk 3 above)
Comment (lewtun) [reply chapters/zh/chapter1/8.mdx:1]: Perfect! I'll merge this now :)

**Pull request merged**